prisma/schema.prisma

```diff
@@ -18,6 +18,8 @@ model User {
   recipes             Recipe[]
   webhooks            Webhook[]
   personalAccessToken PersonalAccessToken[]
+  enabled2FA          Boolean               @default(false)
+  recoveryKeys        String[]
```

Comment:
I would also add a 2fa type column - currently it can be set to e.g. "authenticator" value by default. It will allow us to differentiate between other 2fa types (like SMS) if want to add extra 2fa type in the future.

Reply:
Also I would consider moving 2fa related fields (and perhaps user credentials too?) to a separate table with 1:1 relation. `recoveryKeys` especially are not obvious what are they related to.

Reply:
One more thing: I'm missing 2fa secret. How are we going to validate the user token without knowing the secret?